content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -189,3 +189,21 @@ const userinfo = await prisma.userInfo.findMany({
 ```
 
 Currently, Prisma Client allows you to update a view if the underlying database allows it, without any additional validation.
+
+## Different types of views
+
```

Comment:
We'd normally put some intro text in here rather than leaving a title by itself

Reply:
Suggestions welcome.

Reply:
I've put a suggestion above

Reply:
Ah that makes sense of course!